tools/goawx/types.go

```diff
@@ -688,7 +688,7 @@ type JobEvent struct {
 // User represents an user.
 type User struct {
 	ID              int         `json:"id"`
-	Type            int         `json:"type"`
+	Type            string      `json:"string"`
```

Comment:
Hi @oleghalin ! I think this should be changed back to `json:"type"` as the field in the API is called "type" not "string", right?